app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -432,17 +432,17 @@ function AcuantCapture(
   }
 
   /**
-   * Given a click source, returns a higher-order function that, when called, will log an event
+   * Given a clickSource, returns a higher-order function that, when called, will log an event
    * before calling the original function.
    */
-  function withLoggedClick(source: string, metadata: { isDrop: boolean } = { isDrop: false }) {
+  function withLoggedClick(clickSource: string, metadata: { isDrop: boolean } = { isDrop: false }) {
     return <T extends (...args: any[]) => any>(fn: T) =>
       (...args: Parameters<T>) => {
         if (!isSuppressingClickLogging.current) {
           trackEvent(
             name === 'selfie' ? 'idv_selfie_image_clicked' : `IdV: ${name} image clicked`,
             {
-              source,
+              clickSource,
```

Comment:
This parameter is named and documented in several analytics methods which will need to be updated. Also, there's some inconsistency with how we name parameters that are logged on the backend (camelCase vs. snake_case). Personally, I think this should be snake-case `click_source` to avoid unconventional parameter names on the backend code.

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L838

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L888

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L2083

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L2133

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L4111

https://github.com/18F/identity-idp/blob/97f9d7f09825cdff4dd5def2e8d356da4709f78a/app/services/analytics_events.rb#L4152

Reply:
Lines 838, 2083, 4111 seem to be referencing another source (image source), so I will not adjust those parameter names. Will request review once my changes are pushed (in case my reading is wrong)

Reply:
@aduth I believe I addressed your comments, but I am leaving this thread up since I might have interpreted your input incorrectly.

Reply:
I'm expecting we'd have consistent casing across the board for `click_source`. I'm still seeing a mix of `clickSource` and `click_source`.

Reply:
@aduth I changed to click_source, but now lint is failing in the pipeline. When I run make lint on my local machine I pass (0 errors, 3 ignored warnings). I don't think lint accepts the snake case here. Should I change it back to multiple references (snake case in the BE and camel case in the FE) or should I make them all camel case?

Reply:
To resolve the lint error, I'd suggest naming the function argument in camelCase, but keep the event payload key as snake_case.

i.e.

```ts
function(clickSource: string, ...) {
  trackEvent(
    '...',
    {
      click_source: clickSource,
      // ...
    },
  );
}